pgxn/neon/pagestore_smgr.c

```diff
@@ -1884,10 +1919,12 @@ neon_prefetch(SMgrRelation reln, ForkNumber forknum, BlockNumber blocknum)
 	if (lfc_cache_contains(InfoFromSMgrRel(reln), forknum, blocknum))
 		return false;
 
-	tag = (BufferTag) {
+	BufferTag	tag =
```

Comment:
It would probably be good to compile our C with `-Wdecl-after-statement` similar to what postgres does, but that is a task for another day.

Reply:
It is intentional, see this:
https://github.com/neondatabase/neon/pull/5618#discussion_r1419090374
I'm open to switching to another option, but current seemed to me the preferrable.